test/BIP39Wallet.Tests.cs

```diff
@@ -1,109 +1,98 @@
 using System;
 using System.Text;
-using BIP39Wallet;
 using NBitcoin;
+using NBitcoin.DataEncoders;
 using Xunit;
+// ReSharper disable StringLiteralTypo
 
-namespace BIP39WalletUtils.Tests
+namespace BIP39Wallet.Tests;
+
+public class WalletTests
 {
-    public class WalletTests
+    private const string PubKey = "04c0f6abf0e3122f4a49646d67bacf85c80ad726ca781ccba572033a31162f22e55a4a106760cbf1306f26c25aea1e4bb71ee66cb3c5104245d6040cce64546cc7";
+    [Fact]
+    public void CreateWallet_test()
```

Comment:
What are we testing in this test case?

Reply:
It's a basic functionality test case, to test if the wallet can be created successfully